.github/workflows/upload_image.yml

```diff
@@ -8,10 +8,15 @@ on:
 
 jobs:
   build-specific-architecture:
+    permissions:
+      contents: read
+      # https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#authenticating-to-package-registries-on-github
+      packages: write
```

Comment:
I think we still should set `permissions: read-all`, because the default permission setting of `chaos-mesh/chaos-mesh` to `GITHUB_TOKEN` is "read and write".

Reply:
Makes sense to me. I will roll back the previous commit.

Reply:
Done in [6e3287a](https://github.com/chaos-mesh/chaos-mesh/pull/3700/commits/6e3287a1e5ed36cecace0118ef9bbe488a81e908).